memgpt/openai_tools.py

```diff
@@ -108,7 +108,16 @@ async def wrapper(*args, **kwargs):
 async def acompletions_with_backoff(**kwargs):
     # Local model
     if HOST_TYPE is not None:
-        return await get_chat_completion(**kwargs)
+        retry_count = 5  # Retry up to five times

```

Comment:
@vivi can you take another look at this rq?

Seems like this may not be redundant with the retry loop we have in main, but I'm fine merging this too (though unclear if it will cause more confusion?)